src/Auth0.ManagementApi/Models/TenantChangePassword.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace Auth0.ManagementApi.Models
+{
+    /// <summary>
+    /// Change Password page customization.
+    /// </summary>
+    public class TenantChangePassword
+    {
+        /// <summary>
+        /// true to use the custom change password html, false otherwise (default: false).
+        /// </summary>
+        [JsonProperty("enabled")]
+        public bool Enabled { get; set; }
+
+        /// <summary>
+        /// Replace default change password page with a custom HTML (Liquid syntax is supported).
```

Comment:
Remove "a"

Reply:
Actually like that in the api-docs :(  Will correct upstream too.